ProjectTemplates/templates/WebApi-CSharp/.template.config/template.json

```diff
@@ -134,7 +134,7 @@
       "type": "parameter",
       "datatype": "bool",
       "defaultValue": "false",
-      "description": "Whether to exclude launchSettings.json from the generated template."
+      "description": "Whether to exclude launchSettings.json in the generated template."
```

Comment:
```suggestion
      "description": "Whether to exclude launchSettings.json from the generated template."
```

Reply:
@mmacy : I'll bring that to the ASP.NET Core team (I was synching the templates)